src/util/PrintUtils.java

```diff
@@ -49,23 +53,23 @@ public static void printStatistic(Map<StatisticKey, Long> statistic) {
     }
 
     public static void writeStatistic(Map<StatisticKey, Long> statistic) {
-        String writePath = new File(new File("").getAbsolutePath()).toString() + File.separator + "solver-statistic.txt";
-        StringBuilder sb = new StringBuilder();
+    	JsonObject obj = new JsonObject();
+    	Gson gson = new GsonBuilder().setPrettyPrinting().create();
```

Comment:
Indent the above two lines with one more space to make all lines have the same indentation.